### PR TITLE
fix: Install script output name

### PR DESCRIPTION
**Pull request by m3talsmith**
Issue #, if available: Found in conversation of #569

**Description of Changes**

[//]: #  (A description of the change that you made and the new user experience that it creates)

Fixes problem noticed post merge [here](https://github.com/aws/amazon-genomics-cli/pull/569#discussion_r1036393642)

**Description of how you validated changes**

[//]: #  (A description of you validated your changes and any relevant logs that show your change works)

`make init && make release && ./dist/amazon-genomics-cli/install.sh`

**Checklist**

- [x] If this change would make any existing documentation invalid, I have included those updates within this PR
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [x] I have linted my code before raising the PR
- [x] Title of this Pull Request follows Conventional Commits standard: https://www.conventionalcommits.org/en/v1.0.0/

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: scripts/cli/install.sh [hunk 1]
```diff
@@ -21,15 +21,15 @@ selectCliFile () {
     fileName="agc"
 
     if [[ -z "$archKind" ]]; then
-        eval $filename="agc"
+        eval $fileName="agc"
     else
-        eval $fileName="$filename-$archKind"
+        eval $fileName="$fileName-$archKind"
```
Comment (mattcarp12) [inline scripts/cli/install.sh:26]: I think these should be `fileName="agc"` and `fileName="$fileName-$archKind"`

Comment (m3talsmith) [reply scripts/cli/install.sh:26]: Testing removing the eval, but I do need to keep an eye on POSIX as well. Will have an answer shortly.

File: scripts/cli/install.sh [hunk 2]
```diff
@@ -21,15 +21,15 @@ selectCliFile () {
...
     fi
 
     if [[ "$OSTYPE" == "linux-gnu"* ]]; then
-        eval $1=filename
+        eval $1=$fileName
```
[the remark this answers is not in the document]
Comment (m3talsmith) [reply scripts/cli/install.sh:30]: I'll verify that just incase. Thanks!

**Comment by m3talsmith** [comment]
This shell script is bundled later through the `make release` process. I do believe that we should write some testing around it, but the testing belongs to the make process, rather than this file specifically, as how it gets used varies.

Look for a PR with integration tests around that in a different upcoming PR.

File: scripts/cli/install.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (m3talsmith) [reply scripts/cli/install.sh]: I looked at doing an array supported OSs, but it wasn't worth it. Leaving as is for now.

**Review by mattcarp12 — APPROVED** [review]
Works great

**Review by lingjiekong — APPROVED** [review]

**Pull request merged**